src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,167 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size = 0;
+    private Node first;
+    private Node last;
```

Comment:
```suggestion
    private int size = 0;
    private Node<T> first;
    private Node<T> last;
```

Давай параметризируем Node (это нужно сделать во всех местах программы)